### ISSUE TITLE
Feature: qr only pop up

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#972]
Sockets are notoriously difficult to kill which may interfere with other processes down the line attempting to gain access to the popup port. The solution to this is to have an option to show the qr code only via the lightweight webserver

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.0.5](https://github.com/open-wa/wa-automate-nodejs/compare/3.0.4...3.0.5) (2020-10-20)

- build(deps): bump ts-json-schema-generator from 0.76.1 to 0.77.0 [`#966`](https://github.com/open-wa/wa-automate-nodejs/pull/966)
- Popup no longer hangs on to port after create resolves #972 [`#972`](https://github.com/open-wa/wa-automate-nodejs/issues/972)
- Release 3.0.5 [`a1a20ae`](https://github.com/open-wa/wa-automate-nodejs/commit/a1a20aee9892ce088c85c5855ed328927f64f108)
- Feature: Implemented config `qrPopUpOnly` for qr only pop ups #972 [`83dfbdc`](https://github.com/open-wa/wa-automate-nodejs/commit/83dfbdc2e5e6a4cd3a74c485ecc409c4603b9f8b)
- Updated docs [`e96b655`](https://github.com/open-wa/wa-automate-nodejs/commit/e96b65524a4d3d9ad073e41819e89d5ea4b3acbe)
